README.md

```diff
@@ -272,6 +272,25 @@ every { func() } returns Car() // or you can return mockk() for example
 func()
 ```
 
+### Real partial (relaxed) mock
```

Comment:
I would just call this section "Partial mocking"

Reply:
Had the same feeling, thinking about renaming it to "*calling real method*" instead, but I also like your suggestion. Sold !